curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/intermediate-expressions.md

```diff
@@ -3,6 +3,10 @@ id: 5e7b9f050b6c005b0e76f057
 title: Intermediate Expressions
 challengeType: 11
 videoId: dKgUaIa5ATg
+bilibiliIds:
```

Comment:
are you imbedding this wrong?
<img width="1383" alt="Screen Shot 2021-09-20 at 11 53 53 AM" src="https://user-images.githubusercontent.com/4591597/133977674-72c94637-33b3-4c42-aacb-a67f410760f2.png">

Reply:
I don't think so, no.  I think that's just how they roll, as it were.

Reply:
The flying words are bilibili bullet screen, which allows real-time comments from viewers to fly across the screen like bullets.